operator/pkg/helmreconciler/prune.go

```diff
@@ -43,7 +43,7 @@ import (
 var (
 	// NamespacedResources orders non cluster scope resources types which should be deleted, first to last
 	NamespacedResources = []schema.GroupVersionKind{
-		{Group: "autoscaling", Version: "v2beta1", Kind: name.HPAStr},
+		{Group: "autoscaling", Version: "v2beta2", Kind: name.HPAStr},
```

Comment:
Will this break in 1.26? It seems confusing we use `v2` in some places and `v2beta2` in others

Reply:
this should be v2, forgot to update